tests/calc/test_thermo.py

```diff
@@ -12,9 +12,9 @@
                         dewpoint_from_relative_humidity, dewpoint_from_specific_humidity,
                         dry_lapse, dry_static_energy, el, equivalent_potential_temperature,
                         exner_function, gradient_richardson_number, InvalidSoundingError,
-                        isentropic_interpolation, isentropic_interpolation_as_dataset, lcl,
-                        lfc, lifted_index, mixed_layer, mixed_layer_cape_cin, mixed_parcel,
-                        mixing_ratio, mixing_ratio_from_relative_humidity,
+                        isentropic_interpolation, isentropic_interpolation_as_dataset,
+                        k_index, lcl, lfc, lifted_index, mixed_layer, mixed_layer_cape_cin,
```

Comment:
`isort` wants the imports to look like:
```suggestion
                        isentropic_interpolation, isentropic_interpolation_as_dataset, k_index,
                        lcl, lfc, lifted_index, mixed_layer, mixed_layer_cape_cin,
```

Another option is to install `isort` in your environment and run `isort src tests` before committing and pushing.